### PR TITLE
Adds the ChemMaster 4000

### PR DESCRIPTION
**Pull request by ike709**
image

Everything a chemist needs to make pills and bottles. Also, adds pills and bottles. Patches will come at some point in a future PR.

Pill reagents do get added to the stomach, but I'm unsure if any reagents actually do anything when metabolized yet.

I didn't map it in yet because I didn't want to create conflicts with disposals mapping.

### CODE REVIEW

**Comment by ike709** [comment]
I dunno what to do about this: 
```
##[error]GameObjects/Components/Chemistry/ChemMaster/SharedChemMasterComponent.cs(71,20): error CS8618: Non-nullable field 'id' is uninitialized. Consider declaring the field as nullable. [/home/runner/work/space-station-14/space-station-14/Content.Shared/Content.Shared.csproj]
```

File: Content.Shared/GameObjects/Components/Chemistry/ChemMaster/SharedChemMasterComponent.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    else
+                    {
+                        id = _id;
+                    }
+
+                    isBuffer = _isBuffer.GetValueOrDefault();
+                }
+                else
+                {
+                    pillAmount = _pillAmount.GetValueOrDefault();
```
Comment (DrSmugleaf) [inline Content.Shared/GameObjects/Components/Chemistry/ChemMaster/SharedChemMasterComponent.cs:90]: Field is is not initialized in this branch of the if condition.

File: Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+namespace Content.Client.GameObjects.Components.Chemistry
+{
+    /// <summary>
+    /// Initializes a <see cref="ChemMasterWindow"/> and updates it when new server messages are received.
+    /// </summary>
+    [UsedImplicitly]
+    public class ChemMasterBoundUserInterface : BoundUserInterface
+    {
+#pragma warning disable 649
+        [Dependency] private readonly ILocalizationManager? _localizationManager;
```
Comment (DrSmugleaf) [inline Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs:26]:
```suggestion
        [Dependency] private readonly ILocalizationManager _localizationManager = default!;
```

File: Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+
+            if (disposing)
+            {
+                _window.Dispose();
```
Comment (DrSmugleaf) [inline Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs:98]:
```suggestion
                _window?.Dispose();
```

**Comment by ike709** [comment]
Build finally passes.

File: Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+namespace Content.Client.GameObjects.Components.Chemistry
+{
+    /// <summary>
+    /// Initializes a <see cref="ChemMasterWindow"/> and updates it when new server messages are received.
+    /// </summary>
+    [UsedImplicitly]
+    public class ChemMasterBoundUserInterface : BoundUserInterface
+    {
+#pragma warning disable 649
+        [Dependency] private readonly ILocalizationManager _localizationManager = default!;
```
Comment (PJB3005) [inline Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs:26]: You don't need to pull in `ILocalizationManager` as a dependency. You can use the `Loc` static instead.

**Comment by ike709** [comment]
Improved the UI:

image

**Pull request merged**